src/components/Header.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import colors from "../styles/colors";
+
+const BackgroundColor = styled.div`
+  background-color: ${colors.black};
+`;
+
+const HeaderContainer = styled.div`
+  font-family: "Space Mono", monospace;
+  font-size: 16px;
+  position: relative;
+  padding-top: 131px;
+  padding-bottom: 57px;
+
+  margin-left: 6vw;
+  color: ${colors.white};
+  background-color: ${colors.black};
+`;
+
+const Logo = styled.div`
+  position: absolute;
+  background-color: ${colors.black};
+`;
+
+const NavSection = styled.div`
+  background-color: ${colors.black};
+  width: 100vw;
+`;
+
+const NavCTA = styled.div`
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-top: 13px;
+  padding-bottom: 13px;
```

Comment:
simplify to `padding: 13px 15px` - might also be better to put these into `ems` for consistency